libs/MVS/CMakeLists.txt

```diff
@@ -1,24 +1,4 @@
 
-# Find required packages
```

Comment:
You MUST minimize your changes. Why did you remove that?

Reply:
I modified this file only to delete things that I had added earlier (undoing my damages...) I had intended to restore it to the state that it was originally in, before branching from Phil's branch, so that my only changes are to the top level CMakeLists.txt. If this works just fine, however, I can leave it